bin/reenable_repo_workflows.rb

```diff
@@ -22,7 +22,8 @@ def enable_repo(github, repo_name, workflow_url, workflow_id, dry_run: false, **
 
 github = ManageIQ::Release.github
 
-ManageIQ::Release.github_repo_names_for("ManageIQ").each do |repo_name|
+repos = (ManageIQ::Release.github_repo_names_for("ManageIQ") << "ManageIQ/rbvmomi2").sort
```

Comment:
If we end up having more of these `forked_repos = [...]` might make sense, also more clear why we're adding extras

Reply:
Oh yeah that's a good idea - I have to think how to store that somewhere.